onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc

```diff
@@ -45,7 +45,7 @@ void TestConvTransposeOp(const ConvTransposeOpAttributes& attributes,
     test.AddInput<float>(szNames[i], input_shapes[i], inputs[i]);
   }
   test.AddOutput<float>("Y", expected_output_shape, expected_output);
-  test.Run(expect_result, err_str);
+  test.Run(expect_result, err_str, {kTensorrtExecutionProvider});
```

Comment:
ditto. please try to add more comments broadly explaining why certain tests are disabled.